aten/src/ATen/native/TensorShape.cpp

```diff
@@ -432,15 +471,15 @@ static Tensor cat_sparse(TensorList tensors, int64_t dim) {
           t._values().options().layout_opt(),
           t._values().options().device_opt(),
           t._values().options().pinned_memory_opt());
-      vals_pieces.push_back(native::cat({z1, t._values(), z2}, values_dim));
+      vals_pieces.push_back(at::cat({z1, t._values(), z2}, values_dim));
       idxs_pieces.push_back(t._indices());
     }
     auto sizes_copy = sizes.vec();
     sizes_copy[wrapped] = total_size;
     // This can create an uncoalesced tensor
     return native::sparse_coo_tensor(
-        native::cat(idxs_pieces, 1),
-        native::cat(vals_pieces),
+        at::cat(idxs_pieces, 1),
+        at::cat(vals_pieces),
```

Comment:
extra dispatcher trip seems ok to me, since we're re-using this kernel for SparseCPU + SparseCUDA